vars/terraform.groovy

```diff
@@ -98,7 +98,17 @@ def call(userConfig = [:]) {
               ]) {
                 scmOutput = getInfraSharedTools(sharedToolsSubDir)
 
+                // Retrieve published reports for idempotency (otherwise Terraform's "local_file" will be marked as always re-created as ignored from the SCM)
+                // Note: always run after call to getInfraSharedTools() (or it will be overridden)
+                if(finalConfig.publishReports && finalConfig.publishReports.size > 0) {
+                  for (int i = 0; i < finalConfig.publishReports.size; i++) {
+                    final String fileRelPath = finalConfig.publishReports[i]
+                    file_text = new URL ("https://reports.jenkins.io/${fileRelPath}").getText()
+                    writeFile(file: fileRelPath, text: file_text)
+                  }
+                }
                 try {
+                  sh 'ls -l jenkins-infra-data-reports/azure-net.json'
```

Comment:
What is this `ls` for?

It's not explained in the PR body and its goal isn't immediate (to me at least).

Reply:
Leftover of the debugs, good catch. I've removed it.